macro/build_geometry.C

```diff
@@ -141,7 +142,7 @@ void build_geometry(FairRunSim* run = nullptr)
 
   // beam pipe
   if (isActivated("PIPE")) {
-    run->AddModule(new o2::passive::Pipe("PIPE", "Beam pipe"));
+    run->AddModule(new o2::passive::Pipe("PIPE", "Beam pipe", 1.6f, 0.05));
```

Comment:
what are these new parameters? If specific to upgrade --> needs a protection.

Reply:
Indeed.